crates/node/p2p/src/gossip/handler.rs

```diff
@@ -98,9 +97,9 @@ impl BlockHandler {
     /// Creates a new [BlockHandler] and opens a channel
     pub fn new(
         chain_id: u64,
-        unsafe_recv: watch::Receiver<Address>,
-    ) -> (Self, Receiver<OpNetworkPayloadEnvelope>) {
-        let (sender, recv) = channel();
+        unsafe_recv: Receiver<Address>,
+    ) -> (Self, tokio::sync::mpsc::Receiver<OpNetworkPayloadEnvelope>) {
+        let (sender, recv) = channel(256);
```

Comment:
let's avoid magic numbers

Reply:
gossip isn't even working rn so trying to fix that first before small improvements

Reply:
aka don't want to spend too much time refactoring these parameters before it works